public/app/core/components/TransformersUI/configFromQuery/ConfigFromQueryTransformerEditor.tsx

```diff
@@ -130,15 +130,15 @@ Output (Same as Input[0] but now with config on the Value field)
 | 1626178119127 | 10                      |
 | 1626178119129 | 30                      |
 
-As you can see each row in the source data becomes a separate field. Each field now also has a max 
-config option set. Options like min, max, unit and thresholds are all part of field configuration 
-and if set like this will be used by the visualization instead of any options manually configured 
+As you can see each row in the source data becomes a separate field. Each field now also has a max
+config option set. Options like min, max, unit and thresholds are all part of field configuration
+and if set like this will be used by the visualization instead of any options manually configured
 in the panel editor options pane.
 
 ## Value mappings
 
-You can also transform a query result into value mappings. This is is a bit different as here every 
-row in the config query result will be used to define a single value mapping row. See example below. 
+You can also transform a query result into value mappings. This is is a bit different as here every
+row in the config query result will be used to define a single value mapping row. See example below.
```

Comment:
```suggestion
You can also transform a query result into value mappings. This is is a bit different because every
row in the configuration query result is used to define a single value mapping row. See the following example.
```